source/install.txt

```diff
@@ -71,8 +71,26 @@ Procedure
 
    .. tab::
       :tabid: linux
+
+      Select your desired Linux installation package from the tabs
+      below:
+
+      .. tabs::
+      
+         .. tab:: .tgz
+            :tabid: tgz
       
-      .. include:: /includes/steps/install-shell-generic-linux.rst
+            .. include:: /includes/steps/install-shell-linux-generic.rst
```

Comment:
The "Extract" step should list the TGZ file itself as well, as in:

`tar -zxvf mongosh-0.4.0-linux.tgz`

or 

`tar -zxvf /path/to/mongosh-0.4.0-linux.tgz`